x/liquidstaking/spec/02_state.md

```diff
@@ -88,23 +104,27 @@ This will be consumed at **Handle Queued Withdraw Insurance Requests** when Epoc
 
 **NetAmount** is the sum of the following items
 
-- reward module account’s native token(e.g. `acanto`) balance
-- sum of all chunk balance
-    - The chunk balance will only be as much as the balance accumulated from delegation rewards between epochs. At the end of each epoch, the cumulated chunk balance will be transferred to the reward module account.
+- **reward module account’s native token(e.g. `acanto`) balance**
```

Comment:
Since it has been decided to deduct the `discount_rate` from the remaining del rewards, we can consider deducting the same here

Reply:
I think it would be good to leave this value as is.
The arbitrager will check this value and discount rate also to decide whether they claim or not.
Showing it as is seems more in line with the purpose of the field.

Reply:
I mean, It is not for the purpose of showing, but for the definition of NetAmount, the discount_rate is deducted in del rewards, but if not excluded in the reward pool balance, the consistency is broken and the exchange rate is affected

Reply:
Not sure what is the right way to deal this value. Let me share this topic on the Slack.